### DISCUSSION TITLE
Wrong output when entering decimal value in inputNumber mode="decimal"

### DISCUSSION THREAD

**Discussion by Ranga-Chanakya-Gade** [primefaces/community#1712] [PrimeNG] [answered]
When entering 0.1234 in inputNumber , it is correctly displayed as 0.1234, whereas when I start entering .1234 (instead of 0.1234), the text is displayed as 1234.00. Is there a way to fix this??

Can replicate it in https://primeng.org/inputnumber also.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, this behavior is not supported. You need to type '0.123'. However, you can create a PR to make this possible.
